pkg/sql/parser/sql.y

```diff
@@ -4828,6 +4828,10 @@ constraint_elem:
       Actions: $10.referenceActions(),
     }
   }
+| EXCLUDE USING error
```

Comment:
do we want to try and parse this "with gist etc" or does putting error here catch all of that ?

Reply:
the error catches everything!